Sources/Tracing/Tracer.swift

```diff
@@ -17,15 +17,16 @@ import Instrumentation
 /// An `Instrument` with added functionality for distributed tracing. Is uses the span-based tracing model and is
 /// based on the OpenTracing/OpenTelemetry spec.
 public protocol Tracer: Instrument {
-    /// Start a new `Span` within the given `BaggageContext` at a given timestamp.
+    /// Start a new `Span` with the given `Baggage` at a given timestamp.
+    ///
     /// - Parameters:
     ///   - operationName: The name of the operation being traced. This may be a handler function, database call, ...
-    ///   - context: The carrier of a `BaggageContext` within to start the new `Span`.
+    ///   - baggage: The `Baggage` providing information on where to start the new `Span`.
     ///   - kind: The `SpanKind` of the new `Span`.
     ///   - timestamp: The `DispatchTime` at which to start the new `Span`.
     func startSpan(
         named operationName: String,
-        context: BaggageContextCarrier,
+        baggage: Baggage,
         ofKind kind: SpanKind,
         at timestamp: Timestamp
     ) -> Span
```

Comment:
Good, and we might want an overload accepting `BaggageContext` 👍

Reply:
Good idea. I'll create an issue for it.